test/e2e/framework/ingress_utils.go

```diff
@@ -722,8 +731,8 @@ func (cont *GCEIngressController) deleteStaticIPs() error {
 	return nil
 }
 
-// gcloudList unmarshals json output of gcloud into given out interface.
-func gcloudList(resource, regex, project string, out interface{}) {
+// gcloudComputeResourceList unmarshals json output of gcloud into given out interface.
+func gcloudComputeResourceList(resource, regex, project string, out interface{}) {
```

Comment:
These gcloud funcs are ugly and removing them in this PR would be time consuming. Reckon I create an issue about consuming the cloud provider pkg instead.